### ISSUE TITLE
Upgrade instructions

### ISSUE THREAD

**Issue by samselikoff** [DockYard/ember-in-viewport#171]
Hi there!

Came here because I'm upgrading major versions and I was looking for what the reason for the 2 -> 3 release was. I couldn't find a Changelog but I found [the 3.x release](https://github.com/DockYard/ember-in-viewport/releases/tag/3.0.0) which referenced some PRs.

Are there any breaking changes that we need to be aware of going from 2.x to 3.x?

**Comment by snewcomer** [comment]
Nope!  Same API and general perf improvements.  Will put up another release doc today.

**Comment by snewcomer** [comment]
@samselikoff Here are some release notes.  The thing that triggered 2 -> 3 was updating this library without jQuery and updating lots of deps.  Lmk if you have any questions!

https://github.com/DockYard/ember-in-viewport/releases/tag/3.2.0

**Comment by samselikoff** [comment]
Cool! Maybe in the 3.0 release (since there's no Chnagelog) we could add something like "Upgrade notes: none"? and basically what you just said above?

Btw thanks for the continued work on this addon, it's one of my absolute favs! 🙏
